README.md

```diff
@@ -93,6 +93,13 @@ godot_init!(init);
 
 ### Further examples
 
+
+> ### **IMPORTANT NOTE**
+>
+> Before launching the examples in the godot editor, you must first run `cargo build` and wait for the build operations to successfully.
+>
+> If the editor is opened before the the expected library binary is finished compiling, Godot will fail to load the binary in the editor. When this happens, the editor also does not load any properties or signals associated with the missing NativeScripts in the scene. This will cause the scene tree to be non-functional for any sample that relies on the properties or signals configured in the editor.
```

Comment:
The first sentence is a bit awkward to read. Suggestion:

*At startup, the Godot editor tries to load all resources used by the project, including the native binary. If it isn't present*, the editor *skips* any properties or signals associated with the missing NativeScripts in the scene. This will cause the scene tree to be non-functional for any sample that relies on ~~the~~ properties or signals configured in the editor.